benches/parser.rs

```diff
@@ -0,0 +1,60 @@
+use criterion::{criterion_group, criterion_main, Criterion};
+use std::{fs, path::PathBuf, str::FromStr};
+
+fn benchmark_quil_corpus(c: &mut Criterion) {
+    from_corpus().iter().for_each(|cfg| {
+        c.bench_function(&cfg.name, |b| {
+            b.iter(|| {
+                let _ = quil_rs::Program::from_str(&cfg.program);
+            })
+        });
+    })
+}
+
+struct QuilBenchConfig {
+    name: String,
+    program: String,
+}
+
+fn from_corpus() -> Vec<QuilBenchConfig> {
```

Comment:
`from_corpus`? 👀

Reply:
Haha a "corpus" as in: 

_noun_

1. a collection of written texts, especially the entire works of a particular author or a body of writing on a particular subject.